recipes-kernel/linux/linux-mainline/0010-sun8i-h2-plus-orangepi-zero.dts-disable-mmc1.patch

```diff
@@ -0,0 +1,26 @@
+From 2060e08623dd97b47f5c03651ac865d2efbd203a Mon Sep 17 00:00:00 2001
+From: Pawel Langowski <pawel.langowski@3mdeb.com>
+Date: Tue, 29 Oct 2024 09:47:51 +0100
+Subject: [PATCH] Disable wifi chip
+
```

Comment:
It would be great if we develop a habit of adding some more context on **why** certain changes are introduced.

In this particular case, it could be added in the patch itself, or in the commit text. We've even had some relevant discussion in GH MR that could be linked there as a reference to this explanation. It does not have to be long, just a sentence or two can tremendously improve understanding of commits if we go back to them after a longer time period.

Reply:
[b089531](https://github.com/3mdeb/meta-rte/pull/73/commits/b089531c326b298eb1392b134a70cf4768ad12dc)

Reply:
Thanks